### ISSUE TITLE
RPM Spec file 

### ISSUE THREAD

**Issue by nealef** [mono/mono#9057]
Building an RPM on s390x fails because the %files inventory is looking for files with .pdb extension but s390x produces .mdb files. Could the spec file use .*db in the %files specifications?

Also, as the BE platforms don't have a working csc yet the following fudge may be useful:

+%ifarch s390x
+pushd %{buildroot}%{_prefix}/lib/mono/4.5
+mv csc.exe csc.exe.le
+ln -s mcs.exe csc.exe
+popd
+%endif
+

Neale

**Comment by directhex** [comment]
The .spec file we actually use is here: https://github.com/mono/linux-packaging-mono/blob/centos/mono-core.spec

Should really get rid of the old one from mono.git

**Comment by nealef** [comment]
It has the .pdb file specifications. I believe this is where I grabbed it from and used for the s390x build.

**Comment by directhex** [comment]
Right, right, mdb from mcs vs pdb from csc. Using *db is _probably_ OK. I'm not keen on linking mcs.exe as csc.exe, since behaviour differs

**Comment by nealef** [comment]
True, I should just #ifnarch s390x in the %files section to exclude it.
